src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <Tree :nodes="data" />
+  <Tree :nodes="data" :useCheckbox="true" />
```

Comment:
I think it would be more correct to write the given `useCheckbox` prop with `kebab-case`.

Reply:
Actually every prop and event names in the template should change to `kebab-case`. It seems like there is no consistency for now.

Reply:
You're right. We can resolve this under a separate issue.

Reply:
Okay. I think i can handle this.